src/requests/addLinkToGroup.spec.js

```diff
@@ -37,5 +37,6 @@ test('If genius replies with 200 but LinkResponses contains an error message, th
       }]
     })
 
-  await t.throws(addLinkToGroup(LINK, GROUP_ID), 'Genius API could not generate a code, but still came back with 200')
+  const error = await t.throwsAsync(addLinkToGroup(LINK, GROUP_ID))
+  t.truthy(error instanceof AddLinkToGroupFailed)
```

Comment:
I would suggest you use `t.true` here -- as `instanceof` returns a boolean.

But in this case, I would also suggest passing `AddLinkToGroupFailed` directly to `throwsAsync` as the second argument.

Reply:
ok thanks!